Lib/test/test_webbrowser.py

```diff
@@ -461,11 +461,25 @@ def test_parse_args_error(self):
             "https://example.com --new-window --new-tab",
             "https://example.com -n --new-tab",
             "https://example.com --new-window -t",
-            # Ensure ambiguous shortening fails
-            "https://example.com --new",
         ]:
+            with support.captured_stderr() as stderr:
+                with self.assertRaises(SystemExit):
```

Comment:
You can use `with support.captured_stderr() as stderr, self.assertRaises(SystemExit):` and not touch the indentation.